test/unit/rules/simple-modifiers-test.js

```diff
@@ -0,0 +1,38 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'simple-modifiers',
+
+  config: true,
+
+  good: [
```

Comment:
Can you add some valid test cases to ensure that the things that are disallowed with modifiers are still allowed with other elements/helpers? We need to make sure this rule limits its affects to modifiers only.

Reply:
Added a few other examples covering other helpers as well as custom modifiers that do not use the `modifier` helper explicitly.